cmd/server-main.go

```diff
@@ -482,6 +483,14 @@ func serverMain(c *cli.Context) {
 		fatalIf(errors.New("empty directory"), "Configuration directory cannot be empty.")
 	}
 
+	// Disallow relative paths, figure out absolute paths.
```

Comment:
@bala, `configDirAbs`'s scope ends in the if-else block. To avoid that `configDirAbs` should be defined earlier, defeating the point of restricting its scope.